src/Core/Search/Discussion/Driver/MySqlDiscussionDriver.php

```diff
@@ -9,25 +9,35 @@
  * file that was distributed with this source code.
  */
 
-namespace Flarum\Core\Search\Discussion\Fulltext;
+namespace Flarum\Core\Search\Discussion\Driver;
 
+use Flarum\Core\Discussion;
 use Flarum\Core\Post;
 
-class MySqlFulltextDriver implements DriverInterface
+class MySqlDiscussionDriver implements DriverInterface
 {
     /**
      * {@inheritdoc}
      */
     public function match($string)
     {
-        $discussionIds = Post::where('type', 'comment')
+        $contentDiscussionIds = Post::where('type', 'comment')
             ->whereRaw('MATCH (`content`) AGAINST (? IN BOOLEAN MODE)', [$string])
             ->orderByRaw('MATCH (`content`) AGAINST (?) DESC', [$string])
             ->lists('discussion_id', 'id');
+        $titleDiscussionIds = Discussion::where('is_approved', 1)
+            ->where('title', 'like', "%$string%")
```

Comment:
I believe `LIKE "%$string%"` is going to be *very* slow with anything more than a few dozen thousand discussions.